src/main/java/net/fabricmc/loader/FabricLoader.java

```diff
@@ -333,6 +333,9 @@ protected void addMod(ModCandidate candidate) throws ModResolutionException {
 		ModContainer container = new ModContainer(info, originUrl);
 		mods.add(container);
 		modMap.put(info.getId(), container);
+		for (String alias : info.getAliases()) {
+			modMap.put(alias, container);
+		}
```

Comment:
Am I just missing something or is there no duplication checking? Lets say 2 mods define the same aliases this should fail in a similar way to duplicate mods.

Reply:
I have now implemented the duplication checks, same way as the mods itself does it with a slightly different message ("mod id" -> "alias" of course)